libtransmission/rpcimpl.c

```diff
@@ -1241,6 +1277,11 @@ static char const* torrentSet(tr_session* session, tr_variant* args_in, tr_varia
             }
         }
 
+        if (errmsg == NULL && tr_variantDictFindList(args_in, TR_KEY_labels, &files))
```

Comment:
Either introduce third variable above or merge them all into one with more generic name. Using `files` to store labels isn't nice.